.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,49 @@
+name: Run Tests
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  check-and-test:
+    name: Check and Test
+    runs-on: ubuntu-20.04
+
+    strategy:
+      matrix:
+        mongoVersion: [stable, unstable]
```

Comment:
I'm surprised that this work

Reply:
It's the environment variable - `MONGODB_VERSION` (https://github.com/mongodb-js/compass-crud/pull/317/files#diff-7314d0ebbd2e9537ae4889316745b4fd2fa43cb86275c9caae18a86ba228b642R18). At least that's what it seems from the previous config...